services/todo-legacy/index.ts

```diff
@@ -0,0 +1,35 @@
+import { Application, Server } from '@loopback/core';
+import { setBindings } from './util/setup';
+
+class TodoApplication extends Application {
+  private _startTime: Date;
+
+  constructor() {
+    super();
+    const app = this;
+    setBindings(app);
+  }
+
+  async start() {
+    this._startTime = new Date();
+    const server = new Server(this, { port: process.env.PORT || 3000 });
```

Comment:
FWIW, I think we should eventually move the code handling `process.env.PORT` to a shared module, so that all LoopBack.next applications recognize commonly used ENV variables. See [loopback-boot](https://github.com/strongloop/loopback-boot/blob/39b885f9fb938a04240e7ac8562991bde6593d03/lib/executor.js#L110-L150).